pkg/utils/api/image.go

```diff
@@ -90,36 +88,31 @@ func extract(obj interface{}, path []string, keyPath, valuePath string, fields [
 		}
 		value, ok := output[valuePath].(string)
 		if !ok {
-			return fmt.Errorf("invalid value")
+			// the image may not be present
+			return nil
```

Comment:
Can we log out a message here?

Reply:
Hmmm, we don't have a logger at hands :(

Reply:
@realshuting i used the global logger 🤷